_posts/2020-10-01-annual-report-part-3.md

```diff
@@ -0,0 +1,68 @@
+title: OLS Project and Community report - Part 3
+authors:
+- bebatut
+- malvikasharan
+- yochannah
+image: https://images.unsplash.com/photo-1499244571948-7ccddb3583f1?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2089&q=80
+photos:
+  name: Ross Findon, Unsplash id: @rossf
+  license: CC-BY
+  url: https://unsplash.com
+---
+
+# Year one of Open Life Science (OLS) - Part 3
+
+*This is the last part of the 3 parts OLS project and community report: **Where we want to go***
+
+## Shifting “volunteer culture” of Open Science
+
+The value we create in OLS is impossible without the hard work of our participants (co-founders/lead trainers, mentors, expert guest speakers and project leads), who engage with OLS beyond their usual responsibilities. The first cohort was run entirely on their volunteer-work and for the current cohort we are actively seeking funding to support involvement of our participants in Open Science. 
+
+We teach self-care and mental health awareness as part of OLS, but without the resources to sustain us, it’s very hard to actually practice what we preach. In 2021, we want to run two cohorts with a fairer reward system that empowers our volunteer members and ensures they are able to contribute to Open Science in a sustained manner. We would like for them to receive an honorarium or small support stipend. 
+We would also like to offer small-scale funding to our mentees from low-income backgrounds by offering them access to basic logistics that can improve their experience. For example, headsets, broadband, and other supplies will ensure equitable access to our calls and additional incubator funding for projects. Often a “sticker and biscuits” fund can be incredibly impactful, paying the small expenses needed to kick off a new organisation, whether this is an organisation’s legal registration, support to buy domain names, Zoom rooms, or swag as thank-yous for your community members.
+
+This positive culture change is necessary to prevent unnecessary emotional burden and unfair treatment of contributing members in Open Science caused by unpaid volunteer work.
+
+### OLS curriculum: *not just ‘Open’, but ‘community-minded’*
+
+From the beginning, OLS has been encouraging interested applicants to design their project ideas to not just be ‘Open’ but community-minded. Documentation, reproducibility, transparent reporting and public engagement aspects are strongly integrated into our curriculum and we accept projects with both technical and non-technical scopes. In the first round, many leads of technical projects also reported that their work benefited from the community-mindset we teach in OLS. We intend to strengthen our curriculum to add technical skill-up training (as we currently do for version control), but continue to run the current and next cohorts of the program to support community-oriented projects. We hope this will create momentum in the communities that can benefit immensely from Open Science practices.
```

Comment:
```suggestion
From the beginning, OLS has been encouraging interested applicants to design their project ideas to not just be 'Open' but community-minded. Documentation, reproducibility, transparent reporting and public engagement aspects are strongly integrated into our curriculum and we accept projects with both technical and non-technical scopes. In the first round, many leads of technical projects also reported that their work benefited from the community-mindset we teach in OLS. We intend to strengthen our curriculum to add technical skill-up training (as we currently do for version control), but continue to run the current and next cohorts of the program to support community-oriented projects. We hope this will create momentum in the communities that can benefit immensely from Open Science practices.
```